app/modules/tmr.c

```diff
@@ -60,6 +60,10 @@ tmr.softwd(int)
 #define TIMER_MODE_AUTO 1
 #define TIMER_IDLE_FLAG (1<<7) 
 
+// assuming system_timer_reinit() has *not* been called
+static const uint32_t MAX_TIMEOUT = 0x68D7A3; //SDK 1.5.3 limit
+static const char* MAX_TIMEOUT_STR = "Range: 1-6870947";
```

Comment:
You could get away with [Stringification](https://gcc.gnu.org/onlinedocs/cpp/Stringification.html), eg:
```c6870947
#define MAX_TIMEOUT_DEF 6870947  //SDK 1.5.3 limit
static const uint32_t MAX_TIMEOUT = MAX_TIMEOUT_DEF;
static const char* MAX_TIMEOUT_STR = "Range: 1-" #MAX_TIMEOUT_DEF;
```